timelock-impl/build.gradle

```diff
@@ -18,7 +18,6 @@ dependencies {
     }
 
     compile group: 'com.github.rholder', name: 'guava-retrying'
```

Comment:
I think you can take out Guava-Retrying as well, IIRC I only used this to retry some of the Atomix operations in a controlled way.